package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "flagparser",
+  "version": "1.0.0",
+  "description": "We are building a feature-rich flag parser.",
+  "main": "index.js",
+  "scripts": {
+    "start": "node build/src/index.js",
+    "dev": "npx ts-node src/index.ts",
+    "build": "npx tsc",
+    "watch": "npx tsc -w",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "lint": "gts lint",
+    "clean": "gts clean",
+    "compile": "tsc",
+    "fix": "gts fix",
+    "prepare": "npm run compile",
+    "pretest": "npm run compile",
+    "posttest": "npm run lint"
+  },
+  "keywords": [
+    "flagparser",
+    "parse command line flags"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/imabp/flagparser.git"
+  },
+  "engine": {
+    "node": ">=6.0.0"
+  },
+  "author": "ntsagar9@gmail.com",
```

Comment:
we should remove author for now. I will add this when we publish the package to npm.